xlators/nfs/server/src/mount3.c

```diff
@@ -4414,6 +4414,9 @@ mnt1svc_init(xlator_t *nfsx)
         goto err;
     }
 
+    if (options)
```

Comment:
Here `options` will never be `NULL`. However, `options` may be `NULL` in the `err` label path and should be unreferenced there.

Reply:
done